### DISCUSSION TITLE
Topology 人机交互界面 数据来源有文档说明吗?

### DISCUSSION THREAD

**Discussion by densen2014** [dotnetcore/BootstrapBlazor#320] [Q&A] [answered]
https://www.blazor.zone/Topologies

![image](https://user-images.githubusercontent.com/8428709/212746095-15b412cd-d25a-4196-9e66-5f1a80cb8405.png)

组件说明有点少, 想详细学习一下

**Comment by ArgoZhang** [comment, marked as answer]
稍后详细完善文档，这个是 le5le 的一个开源引擎做驱动的，需要对 le5le 的有一些了解
